src/basic/time-util.h

```diff
@@ -112,10 +113,15 @@ static inline bool triple_timestamp_is_set(const triple_timestamp *ts) {
 usec_t triple_timestamp_by_clock(triple_timestamp *ts, clockid_t clock);
 
 usec_t timespec_load(const struct timespec *ts) _pure_;
+_pure_ static inline usec_t statx_timestamp_load(const struct statx_timestamp *ts) {
+        return timespec_load(&(const struct timespec) { .tv_sec = ts->tv_sec, .tv_nsec = ts->tv_nsec });
+}
 nsec_t timespec_load_nsec(const struct timespec *ts) _pure_;
+_pure_ static inline nsec_t statx_timestamp_load_nsec(const struct statx_timestamp *ts) {
+        return timespec_load_nsec(&(const struct timespec) { .tv_sec = ts->tv_sec, .tv_nsec = ts->tv_nsec });
+}
```

Comment:
these helpers make a ton of sense, but I think they should be defined in stat-util.h, somewhere close to statx_fallback() or STRUCT_NEW_STATX_DEFINE()...

After all these are more helpers for statx() than for timespec.